.github/workflows/virus-scan.yml

```diff
@@ -0,0 +1,97 @@
+name: Virus scan
+on:
+  release:
+    types: [published]
+jobs:
+  virus-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install ClamAV
+        run: |
+          sudo apt-get update && sudo apt-get install clamav
+          clamVersion=$(clamscan --version)
+          echo $clamVersion
+          echo "CLAMAV_VERSION=$clamVersion" >> $GITHUB_ENV
+      - name: Update virus signature database
+        run: |
+          sudo systemctl stop clamav-freshclam
+          sudo freshclam
+          sudo systemctl start clamav-freshclam
+      - name: Get release
+        uses: actions/github-script@v3.1.1
+        with:
+          github-token: ${{secrets.RELEASE_ANTIVIRUS_GITHUB_ACCESS_TOKEN_PBOT4}}
+          script: |
+            const fs = require('fs');
+            
+            await io.mkdirP('github-release-assets');
+            
+            let release = await github.repos.getReleaseByTag({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              tag: '${{ github.event.release.name }}'
+            });
+            
+            core.exportVariable('RELEASE_ID', release.data.id);
+            core.exportVariable('RELEASE_BODY', release.data.body);
+            core.exportVariable('RELEASE_HTML_URL', release.data.html_url);
+            
+            for (const assetInfo of release.data.assets) {
+              let asset = await github.request(assetInfo.browser_download_url);
+              await fs.writeFile('github-release-assets/' + assetInfo.name, Buffer.from(asset.data), () => {});
+            }
+            
+            let zipball = await github.request(release.data.zipball_url);
+            await fs.writeFile('github-release-assets/source.zip', Buffer.from(zipball.data), () => {});
+            
+            let tarball = await github.request(release.data.tarball_url);
+            await fs.writeFile('github-release-assets/source.tar.gz', Buffer.from(tarball.data), () => {});
+            
+      - name: Run ClamAV
+        # Don't automatically fail on first non-zero return code by skipping -e parameter
+        # May highlight as error but docs say is valid: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#custom-shell
+        shell: "/usr/bin/bash {0}"
+        run: |
+          sudo clamscan --infected github-release-assets/ > scan-results.log
+          echo "CLAMAV_RETURN_CODE=$?" >> $GITHUB_ENV
```

Comment:
As far as I remember we wouldn't need to go over the env variables necessarily. You can reference output variables of previous steps in consecutive steps without having to resort via env

Reply:
```
- name: Run ClamAV
  id: run-clamav
  run: |
     echo "::set-output name=return_code::$?" 
```

subsequent steps

```
      - name: Notify Slack on viruses detected
        if: ${{ steps.run-clamav.outputs.return_code == '1' }}
```

because outputs are unique by steps you also you don't need to be overly specific with the variable names